components/shared/webrender/rendering_context.rs

```diff
@@ -664,43 +672,34 @@ impl Framebuffer {
 pub struct OffscreenRenderingContext {
     parent_context: Rc<WindowRenderingContext>,
     size: Cell<PhysicalSize<u32>>,
-    back_framebuffer: RefCell<Framebuffer>,
-    front_framebuffer: RefCell<Option<Framebuffer>>,
+    framebuffer: RefCell<Framebuffer>,
 }
 
 type RenderToParentCallback = Box<dyn Fn(&glow::Context, Rect<i32>) + Send + Sync>;
 
 impl OffscreenRenderingContext {
     fn new(parent_context: Rc<WindowRenderingContext>, size: PhysicalSize<u32>) -> Self {
-        let next_framebuffer = Framebuffer::new(parent_context.gl_api(), size);
+        let framebuffer = RefCell::new(Framebuffer::new(parent_context.gleam_gl_api(), size));
         Self {
             parent_context,
             size: Cell::new(size),
-            back_framebuffer: RefCell::new(next_framebuffer),
-            front_framebuffer: Default::default(),
+            framebuffer,
         }
     }
 
     pub fn parent_context(&self) -> &WindowRenderingContext {
         &self.parent_context
     }
 
-    pub fn front_framebuffer_id(&self) -> Option<gl::GLuint> {
-        self.front_framebuffer
-            .borrow()
-            .as_ref()
-            .map(|framebuffer| framebuffer.framebuffer_id)
-    }
-
     pub fn render_to_parent_callback(&self) -> Option<RenderToParentCallback> {
         // Don't accept a `None` context for the read framebuffer.
```

Comment:
```suggestion
        // Don't accept a `None` context for the framebuffer.
```

Reply:
Cool. I've changed this to say "source" for clarity.